GalaxyZooTouchTable/GalaxyZooTouchTable/Models/GalaxyExampleFactory.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Windows.Media.Imaging;
+
+namespace GalaxyZooTouchTable.Models
+{
+    enum GalaxyType
+    {
+        Smooth,
+        Features,
+        Star
+    }
+
+    public class GalaxyExample
+    {
+        BitmapImage ImageOne { get; }
+        BitmapImage ImageTwo { get; }
+        BitmapImage ImageThree { get; }
+        string Title { get; }
+        string Description { get; }
+        string Description2 { get; }
+    }
+
+    static class GalaxyExampleFactory
+    {
+        public static GalaxyExample Create(GalaxyType type)
+        {
+            switch (type)
+            {
+                case GalaxyType.Smooth:
+                    return new SmoothGalaxy();
+                case GalaxyType.Features:
+                    return new FeaturesGalaxy();
+                case GalaxyType.Star:
+                    return new StarGalaxy();
+                default:
```

Comment:
I think it's good to leave the default clause out if you have exhausted all of the possible cases on an enum. That way, your code will fail loudly if you add a new value to `GalaxyType` and don't update this switch statement

Reply:
Ok. Returning `null` here should produce an error if an example isn't present.